src/useWebReader.tsx

```diff
@@ -67,6 +68,8 @@ export default function useWebReader(
   const readerType = getReaderType(
     manifest ? manifest.metadata.conformsTo : null
   );
+  const getContentHtml = getContent as GetContent<string> | undefined;
+  const getContentPdf = getContent as GetContent<Uint8Array> | undefined;
```

Comment:
This isn't ideal, but I think it's okay for right now. In the future I'm thinking we might just want to not have a `useWebReader` and instead only export a `useHtmlReader` and `usePdfReader` separately. Seems like it might be simpler for these kinds of types, and I don't forsee that many cases when you would need to use both simultaneously